projects/packages/forms/src/blocks/label/edit.js

```diff
@@ -17,21 +18,53 @@ const getLabelOrFallback = ( label, placeholder ) => {
 	return label ?? placeholder;
 };
 
-const WithNotchedWrapper = ( { formStyle, children } ) => {
+const OPTIONS_FIELDS = [ 'jetpack/field-single-choice', 'jetpack/field-multiple-choice' ];
```

Comment:
```suggestion
const OPTIONS_FIELDS = [ 'jetpack/field-radio', 'jetpack/field-checkbox-multiple' ];
```

For some reason the folder names differ from the block names.